env/var.go

```diff
@@ -42,6 +43,27 @@ const (
 	DURATION
 )
 
+type FeatureStatus byte
+
+const (
+	Alpha FeatureStatus = iota
```

Comment:
In a world where we canonicalize these Unknown's should become forbidden. Can we add a 'Dev' status

Reply:
Platform env vars are not registered with the env package, and the ctrlz page lists their default values as UNKNOWN.  I don't think we have any concept of 'Dev' status elsewhere in the project.  Would 'experimental' make more sense?